apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -3,88 +3,14 @@ import {connect} from 'react-redux';
 import i18n from '@cdo/locale';
 import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
 import Button from '@cdo/apps/templates/Button';
-import moment from 'moment/moment';
 import PropTypes from 'prop-types';
 import queryString from 'query-string';
 import color from '@cdo/apps/util/color';
 import $ from 'jquery';
-import RubricField from './RubricField';
-import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import {CommentArea} from './CommentArea';
-
-const styles = {
```

Comment:
Just curious, what prevented these styles from being moved automatically?

Reply:
I wondered the same thing! I haven't investigated yet, but already made myself a ticket to take a look https://codedotorg.atlassian.net/browse/LP-1915